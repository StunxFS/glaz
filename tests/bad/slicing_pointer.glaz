fn main() {
	let x: *i32 = &0;
	_ = unsafe { x[1:5] };
}
