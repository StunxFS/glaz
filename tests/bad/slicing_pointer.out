tests/bad/slicing_pointer.glaz:3:16: error: cannot slice a pointer
     | 
   3 | 	_ = unsafe { x[1:5] };
     | 	             ^^^^^^
glaz error: could not compile `slicing_pointer`, aborting due to previous error.
